resources/js/components/Product/AddToCart.vue

```diff
@@ -80,7 +80,10 @@ export default {
 
             try {
                 let response = await window.magentoGraphQL(
-                    `mutation (
+                    config.queries.cart +
+                        `
+
+                    mutation (
```

Comment:
This is part of how fragments are used.
https://graphql.org/learn/queries/#fragments

It may be a good idea to move the fragments to the end of the queries like the examples on the docs.
Would renaming `config.queries.cart` to `config.fragments.cart` improve legibility?